src/components/ActivityButtons/ActivityButtons.js

```diff
@@ -1,25 +1,83 @@
-import React from 'react'
+import React, { useState } from 'react'
 import classes from './ActivityButtons.module.scss'
 import PropTypes from 'prop-types'
+import { DateTime } from 'luxon'
+
+const ActivityButtons = ({ activity, link, suggestActivity, startOver }) => {
+const [ saved, setSaved ] = useState(false)
+  const saveActivity = () => {
+    const activityToSave = {
+      date: DateTime.local().toLocaleString(),
+      description: activity,
+      link: link || null,
+    }
+    addToLocalStorage(activityToSave)
+  }
+  
+  const addToLocalStorage = (activityToSave) => {
+    if (localStorage.getItem('savedActivities') === null) {
+      let activities = [activityToSave]
+      localStorage.setItem('savedActivities', JSON.stringify(activities))
+      setSaved(true)
+    } else {
+      const activities = JSON.parse(localStorage.getItem('savedActivities'))
+      activities.push(activityToSave)
+      localStorage.setItem('savedActivities', JSON.stringify(activities))
+      setSaved(true)
+    }
+  }
 
-const ActivityButtons = ({ activity, suggestActivity, startOver }) => {
   return (
     <>
-      <h3>What if you {activity.toLowerCase()}?</h3>
-      <div className={classes.btnBar}>
-        <button className={classes.activityBtn}>Thanks! I'll try that.</button>
-        {activity.includes("stargazing") && (
-          <button className={classes.activityBtn}>
-            Can we do that together?
+      {!saved && (
+        <>
+          <h3>What if you {activity[0].toLowerCase() + activity.substring(1)}?</h3>
+          {link && 
+          <h4>Check it out 
+            <a href={link} target='_blank' rel="noopener noreferrer">here!</a>
+          </h4>}
+          <div className={classes.btnBar}>
+            <button className={classes.activityBtn} onClick={saveActivity}>
+              Thanks! I'll try that.
+            </button>
+            {activity.includes("stargazing") && (
+              <button className={classes.activityBtn}>
+                Can we do that together?
+              </button>
+            )}
+            <button className={classes.activityBtn} onClick={suggestActivity}>
+              Can you suggest something else?
+            </button>
+            <button className={classes.activityBtn} onClick={startOver}>
+              Can I start over?
+            </button>
+          </div>
+          <br />
+        </>
+      )}
+      {saved && (
+        <>
+          <h3>Ok! I'll add "{activity}" to your activity journal.</h3>
+          <button
+            className={classes.activityBtn}
+            onClick={(event) => {
+              startOver(event)
+              setSaved(false)
+            }}
+          >
+            View My Journal
+          </button>
+          <button
+            className={classes.activityBtn}
+            onClick={(event) => {
+              startOver(event)
+              setSaved(false)
+            }}
+          >
+            Pick Another Activity
```

Comment:
This could potentially be another component, but I'm diggity down for it to be like this.